package.json

```diff
@@ -21,14 +21,31 @@
   "license": "MIT",
   "main": "main.js",
   "types": "main.d.ts",
+  "module": "esm/main.mjs",
```

Comment:
this isn't necessary and will be ignored

more importantly, I just learned that tsc doesn't enforce valid output when you set `node16` unless you also have `"type": "module"`